packages/content-format/src/nodes/index.ts

```diff
@@ -16,4 +16,5 @@ export * from './PlaceholderNode';
 export * from './QuoteNode';
 export * from './StoryBookmarkNode';
 export * from './StoryEmbedNode';
+export * from './Table';
```

Comment:
Let's either rename this file to plural to match `Lists`, or rename `Lists` to singular to match `Table` naming :)